Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -154,22 +156,32 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         return consoleMessageView;
     }
 
-    updatePreviousMessageRepeatCount(count, timestamp)
+    updatePreviousMessageRepeatCount(target, count, timestamp)
     {
-        console.assert(this._previousMessageView);
-        if (!this._previousMessageView)
+        let previousMessageView = this._previousMessageView;
+        console.assert(previousMessageView);
+        if (!previousMessageView)
             return false;
 
-        var previousIgnoredCount = this._previousMessageView[WI.JavaScriptLogViewController.IgnoredRepeatCount] || 0;
-        var previousVisibleCount = this._previousMessageView.repeatCount;
+        if (previousMessageView.message.target._identifier !== target._identifier) {
```

Comment:
i think we can compare `previousMessageView.message.target !== target` directly since we should never have more than one `WI.Target` instance for a given `identifier`

at the very least we should be using `target.identifier` public `get` instead of the private/internal property